src/mozxchannel/git/process.py

```diff
@@ -51,6 +51,8 @@ def loadRevs(self):
         revs = self.target.converted_revs()
         for repo in self.repos:
             repo_name = repo.name
+            if "firefox-android" in repo_name:
+                repo_name += f"/{self.project}"
```

Comment:
Bug: Python doesn't accept the `+=` operator on strings because they're immutable. Here's another way to implement the same functionality: 

```suggestion
                repo_name = f"{repo_name}/{self.project}"
```

Let's change all places where we use `+=` on strings.